devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -101,5 +142,172 @@ protected SpanByte SliceGenericBuffer(int start, int length)
 
             return _genericBuffer;
         }
+
+
+        /// <summary>
+        /// Draws a pixel on the screen.
+        /// </summary>
+        /// <param name="x">The x coordinate on the screen.</param>
+        /// <param name="y">The y coordinate on the screen.</param>
+        /// <param name="useWhite">Indicates if color to be used, default yes unless is inverted screen.</param>
+        public void DrawPixel(int x, int y, bool useWhite = true)
```

Comment:
I feel like `useWhite` should be renamed to `inverted` because at the end of the day, it's about that.

Reply:
ty, that works